### PR TITLE
os-smart: Re-add descriptions for cron actions

### PR DESCRIPTION
**Pull request by skazi0**
The "description" fields were removed to not flood the UI with duplicate
items but some of them are actually useful for cron.
Descriptions for actions related to tests were re-added with unique
content to enable periodic SMART tests via cron.

### CODE REVIEW

File: sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf [hunk 1]
```diff
@@ -57,27 +57,32 @@ command:/usr/local/sbin/smartctl -t offline
...
 type:script_output
 message:Testing device %s (conveyance)
+description:Run SMART test (conveyance)
 
 [abort]
 command:/usr/local/sbin/smartctl -X
 parameters:%s; exit 0
 type:script_output
 message:Abort test on device %s
+description:Abort SMART tests
```
Comment (fichtner) [inline sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf:88]: I'm unsure this makes sense in cron environments, but otherwise ok

**Comment by fichtner** [comment]
Question: how will you inspect the results of these cron runs?

File: sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf (hunk 1 above)
Comment (skazi0) [reply sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf:88]: There are times when SMART tests get stuck. Maybe someone would like to run cleanup once a month or so just to be sure or just abort any long-running test before starting a new one.

**Comment by skazi0** [comment]
@fichtner all of these commands do the actual work in the background. Results of the tests can be checked using existing "View Logs" feature of SMART plugin.

**Comment by skazi0** [comment]
@fichtner any more questions or comments?

**Review by fichtner — APPROVED** [review]

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**